.github/workflows/deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Shopify Analytics Deploy
+
+on:
+  push:
+    branches: [ master ]
+  # Allow running this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  ECR_IMAGE_REPO: "884681002735.dkr.ecr.us-east-1.amazonaws.com/shopify-analytics"
+  DEPLOY_IAM_ROLE: "arn:aws:iam::884681002735:role/github_actions/github-actions-role"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Build Image
+        run: docker build -t ${{ env.ECR_IMAGE_REPO }}:${{ github.sha }} --label org.label-schema.name="shopify-analytics" .
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ env.DEPLOY_IAM_ROLE }}
+          aws-region: us-east-1
+
+      - name: ECR login
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Push to ECR
+        run: docker push ${{ env.ECR_IMAGE_REPO }}:${{ github.sha }}
```

Comment:
We can get the registry directly from the ecr login step and thus we don't have to hardcode the `ECR_IMAGE_REPO` variable.

I think it would be something like `${{ steps.login-ecr.outputs.registry }}`

Reply:
If this was fully automated end-to-end, i'd agree. But we have the URI hardcoded at the consumer: https://github.com/iFixit/airflow-dags/blob/f17de130b8b24e55de248c0a9fb3a101c61eddd6/shopify-analytics/au.py#L40-L41

I like keeping the full URIs around to keep them easier to search for. 

I think eventually we'll want to migrate the Airflow tasks to Dagster, where we can get some better end-to-end CI, and that problem can go away.